api/pre_process_data.py

```diff
@@ -6,58 +6,62 @@
 
 
 class PreprocessData:
+    def __init__(self, from_year="1990"):
+        self.from_year = from_year
+
     def emissionsAndLand(self, country="World"):
-        return self.__preprocess_data(
-            file1="greenhouse_gas_emission",
-            file2="agricultural_land",
+        emission_df = pd.read_csv("data/greenhouse_gas_emission.csv")
+        agricultural_land_df = pd.read_csv("data/agricultural_land.csv")
+        return self.__process_data(
+            df1=emission_df,
+            df2=agricultural_land_df,
             indicator1="Emissions",
             indicator2="Agricultural Land",
             info="emissions_and_land",
             country=country,
         )
 
     def emissionAndCerealYield(self, country="World"):
-        return self.__preprocess_data(
-            file1="greenhouse_gas_emission",
-            file2="cereal_yield",
+        emission_df = pd.read_csv("data/greenhouse_gas_emission.csv")
+        cereal_yield_df = pd.read_csv("data/cereal_yield.csv")
+        return self.__process_data(
+            df1=emission_df,
+            df2=cereal_yield_df,
             indicator1="Emissions",
             indicator2="Cereal",
             info="emissions_and_cereal_yield",
             country=country,
         )
 
     def populationAndArableLand(self, country="World"):
-        return self.__preprocess_data(
-            file1="population",
-            file2="arable_land",
+        population_df = pd.read_csv("data/population.csv")
+        arable_land_df = pd.read_csv("data/arable_land.csv")
+        return self.__process_data(
+            df1=population_df,
+            df2=arable_land_df,
             indicator1="Population",
             indicator2="Arable Land",
             info="population_and_arable_land",
             country=country,
         )
 
-    def __preprocess_data(
-        self, file1, file2, indicator1, indicator2, info, country="World"
-    ):
+    def __process_data(self, df1, df2, indicator1, indicator2, info, country="World"):
         # File names = the name of the file containing data
```

Comment:
Change this to a docstring please.

Use this [format](https://github.com/Khan/style-guides/blob/master/style/python.md#docstrings) (Will be added to the docs, too.)
Example:
```
def google_docstrings(num1, num2
                      ) -> int:
    """Add up two integer numbers.

    This function simply wraps the ``+`` operator, and does not
    do anything interesting, except for illustrating what
    the docstring of a very simple function looks like.

    Args:
        num1 (int) : First number to add.
        num2 (int) : Second number to add.

    Returns:
        The sum of ``num1`` and ``num2``.

    Raises:
        AnyError: If anything bad happens.
    """
    return num1 + num2
```